CONTRIBUTING.md

```diff
@@ -1,126 +1,177 @@
+[DCO GitHub Check](https://github.com/apps/dco) automatically enforces DCO for all commits.
+Contributors are required to ensure that every commit message contains the signature:
+`Signed-off-by: NAME SURNAME <email@address.example.org>`
+The best way to achieve this automatically is to specify the following alias in the `~/.gitconfig` file:
+```.gitconfig
+[alias]
+ci = commit -s
+```
+### Changelog
+The [CHANGELOG](CHANGELOG.md) is automatically generated from Github PRs and Issues during release.
+Use dedicated [keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in PR message or [manual PR and Issue linking](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue) for clean changelog generation.
+Issues and PRs should be also properly tagged with valid project tags ("bug", "enhancement", "won't fix", etc )
 
 ## Documentation
 
-If the contribution changes the existing APIs or user interface it must include sufficient documentation to explain the use of the new or updated feature.
-[CHANGELOG](CHANGELOG.md) is automatically generated from Github PRs and Issues. Please use special [keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to link PR to and Issue for a clean changelog generation.
+If contribution changes the existing APIs or user interface, it must include sufficient documentation explaining the use of the new or updated feature.
+
+## k8gb.io website
 
+k8gb.io website is a Jekyll-based static website generated from project markdown documentation and hosted by GitHub Pages.
+`gh-pages` branch contains the website source, including configuration, website layout, and styling.
+Markdown documents are automatically populated to `gh-pages` from the main branch and should be authored there.
+Changes to the k8gb.io website layout and styling should be checked out from `gh-pages` branch and  PRs should be created against `gh-pages`.
 
-[golang-style-doc]: https://github.com/golang/go/wiki/CodeReviewComments
+### Local website authoring and testing
+
+These instructions will help you to set up and use local website authoring and testing environment:
+- Install Jekyll and Bundler following the [instructions](https://jekyllrb.com/docs/installation/).
+- Create dedicated [GitHub Personal Access Token](https://github.com/settings/tokens/new) with `public_repo` permission.
+- Run the following command to generate and run the local copy of the k8gb.io website:
+  ```sh
+  JEKYLL_GITHUB_TOKEN=<GITHUB_PAT_TOKEN> exec jekyll serve --watch --livereload
+  ```
+  where `<GITHUB_PAT_TOKEN>` is the previously generated GitHub PAT token
+- Open `http://127.0.0.1:4000/` page in your browser
```

Comment:
This one begs for accompanied Make target

Reply:
@ytsarev  I suggest making this in separate PR. There was a suggestion from @k0da in #475 to use docker-based jekyll execution. This needs some tooling investigation and POC, including Makefile target update. Also, either the whole Makefile or just this mentioned target should be merged to the `gh-pages` branch, as website authoring is done against `gh-pages`. 
For now, I'm just describing manual way to run Jekyll tooling locally, which was missing before.